controller/src/main/java/io/pravega/controller/metrics/HostContainerMetrics.java

```diff
@@ -66,20 +66,20 @@ public void updateHostContainerMetrics(Map<Host, Set<Integer>> oldMapping, Map<H
     }
 
     private void reportContainerCountPerHost(Host host, Set<Integer> containerIds) {
-        DYNAMIC_LOGGER.reportGaugeValue(nameFromHost(SEGMENT_STORE_HOST_CONTAINER_COUNT, host.toString()), containerIds.size());
+        DYNAMIC_LOGGER.reportGaugeValue(SEGMENT_STORE_HOST_CONTAINER_COUNT, containerIds.size(), hostTag(host.toString()));
```

Comment:
I see that now metrics are indexed in another way. While this looks more flexible, I have a question: are dashboards impacted by this change? (e.g., Grafana dashboards)

Reply:
Yes, today's hierarchical data model is not sustainable, migrating towards dimensional data model is the direction we have to go; Eron in the demo also said they are willing to accept essential breaking changes.
Having said that, effort has been made to make the migration backwards as much as possible. Metrics names, tags, prefix are kept same. Metric type suffix are removed per the demo feedback.